views/islandora_usage_stats_collections.views_default.inc

```diff
@@ -0,0 +1,140 @@
+<?php
+$view = new view();
```

Comment:
This code (all the default view code, that is) should live in an implementation of [`hook_views_defaul_views()`](https://api.drupal.org/api/views/views.api.php/function/hook_views_default_views/7).